cql3/statements/update_statement.cc

```diff
@@ -191,16 +191,16 @@ insert_prepared_json_statement::execute_set_value(mutation& m, const clustering_
     if (!value) {
         visit(*column.type, make_visitor(
         [&] (const list_type_impl&) {
-            lists::setter::execute(m, prefix, params, column, {});
+            lists::setter::execute(m, prefix, params, column, expr::constant::make_null());
```

Comment:
Right now finding the type of each `NULL` could be hard, because an empty `shared_ptr` means null. A lot of code would have to be changed.

Maybe we can revisit this later when `bind()` is also implemented using expressions, finding the types should be easier then.

Reply:
I hope to get rid of bind(). It doesn't make much sense to bind an expression to its inputs.

Reply:
Maybe not in `bind`, but later when `shared_ptr<term>` is not used anymore.